samples/README.md

```diff
@@ -41,7 +41,7 @@ Key fields for each [`tabular-data-resource`](https://specs.frictionlessdata.io/
 
 | **Field** | **Description** | **Required** |
 | --------- | --------------- | ------------ |
-| `name` | Short sluggable name used to refer to data in this file. | Required |
+| `name` | Short [sluggable](https://en.wikipedia.org/wiki/Clean_URL#Slug) name used to refer to data in this file. `name` must be unique within this datapackage. | Required |
```

Comment:
Is this sluggable, or does it need to be a slug?

Reply:
should be sluggable.